app/models/amazon_agent_coordinator.rb

```diff
@@ -0,0 +1,385 @@
+require 'yaml'
+require 'open3'
+require 'net/scp'
+require 'linux_admin'
+require 'amazon_ssa_support'
+
+class AmazonAgentCoordinator
```

Comment:
This should not be in the models directory...in the new plugin probably ManageIQ::Providers::Amazon::CloudManager::AgentCoordinator or something.